packages/sui-i18n/src/i18n.js

```diff
@@ -7,6 +7,7 @@ export default class Rosetta {
     this._culture = null
     this._currency = null
     this._languages = null
+    this._extendedIntlNumberOptions = null
```

Comment:
how about using some more simple naming closer to what IMHO it actually does?

```suggestion
    this._defaultNumberFormatOptions = null
```

what do you think?

Reply:
Sorry, but i don't agree, IMHO this is values are an extended options which are added to those already received and set in the different methods that use i18n.n, for example i18n.c

```js
  c(number, minimumFractionDigits = 0, culture) {
    return this.n(
      number,
      {
        style: 'currency',
        currency: this._currency,
        minimumFractionDigits
      },
      culture
    )
  }
```